### ISSUE TITLE
Increase monster size for all phases

### ISSUE THREAD

**Issue by burrage** [curiouslearning/FeedTheMonsterJS#1001]
**User Story**
As a UX designer,
I want to increase the size of the monster and its accompanying animations for all phases of the monster,
So that it is more prominently featured in the UI and there is more space for a child to drag a stone to, similar to the Unity version of FTM.

We are looking into doing this programmatically and if that has any ramifications on performance or if we want to take another approach.

Eyeballing, I think 1.5x the size should be equivalent to what the Unity size is roughly.

**Acceptance Criteria**
Given that I am playing FTM,
When I look at the monster and accompanying animations,
Then the monster should appear closer to the size of the monster and animation in the Unity version of FTM.

**Comment by burrage** [comment]
In QA, let's ensure this doesn't throw off where someone can drag a stone.

CC:  #1104

**Comment by Ashish1998** [comment]
@burrage Hiii Ben, for this task we increased the monster size and we tried to test it in few Real Devices and  emulators
We were not able to get a real device with these specs (Android 9 and 2GB). We have few Android 9 device but with more RAM, there we are not facing any issues.
We tried to test in Android Studio, the emulator devices we tried to configure a device with our minimum app requirements (Android 9 & 2GB RAM) but there we are facing an issue the FTM was not downloading, (may be it was an error for the emulator side). 
In few other emulators we only had the option to configure Android version, there we did not face much issues with the animation timing or the performance, but we had some lag in the audios (majorly Feedback audio). 
Please let us know your suggestion on this Ben
Thank you.

**Comment by burrage** [comment]
@Ashish1998 Thanks for the info, Ashish!

For now, let's keep this out of a push to production in wrapping up this sprint CC: @Rajesh1041 .

Is the most up to date version of this on development right now? I'll check it out.

**Comment by amitsinghsutara** [comment]
Hey @burrage ,

Hope you are doing well ;)

Yes Ben we have the latest changes on the dev build along with that we also have updated wrong answer monster animation as well!

**Comment by burrage** [comment]
Hi all, in revisiting this task there're a couple of things:

1.) As a separate task card, we should figure out why we can't play our app on an emulator device that has the minimum device specs. That's probably an issue that will continue to rear its head again and again with regard to testing.

2.) I do have an Android 10 2GB physical device so I don't mind playing this a little later on that device. 

3.) In trying it out previously on a laptop, I noticed that the size of where you can drag the stone did not increase with the monster size, which I imagine will be a problem when I try it on mobile as well.

I'll report back on 2. If @Ashish1998 could spin up a "bug" report regarding the minimum Android device emulator that would be great.

Let's wait on doing anything with 3 because we may just need to come at this from a different angle than we first tried.

**Comment by amitsinghsutara** [comment]
Hey @burrage ,

Currently we have reverted the changes to original ones!

**Comment by burrage** [comment]
Thanks guys, I'll get to testing this on my device now...

**Comment by divyahariharan2103** [comment]
Hi @burrage , Wanted to know if there is any update on this task ?

**Comment by burrage** [comment]
OK so... I'm not noticing any further performance lags in comparison to when the monster was smaller (which is good!). The size of the monster being larger is definitely an improvement.

However, the area for which to drag the stone just feels too small now. If we look back at the Unity version, we can drag the stone basically anywhere on the monster (not just the mouth or egg opening) and it'll count as dragging the stone to the monster.

Let's improve that and give this another test.

**Comment by amitsinghsutara** [comment]
Hey @burrage ,

Hope you are doing well ;)

Thank you for sharing your thoughts,
Ben as per area in which we want to drag the stone over the monster  we will work on increasing  the area and give it another try!

Best Regards 
Amit

**Comment by amitsinghsutara** [comment]
Hey @burrage ,

Status update:- we have increased the drop area of stones,
Please have a look and share your feedback!

Best Regards 
Amit

**Comment by burrage** [comment]
@amitsinghsutara Better than before! I think I want to ship this but while we're working on this still... if you refer back to the Unity version, the user can basically drag the stone to any portion of the monster and it counts. Would that be considerably more effort to do?

**Comment by amitsinghsutara** [comment]
Hey @burrage ,

In recent one we have increased the drop area,
Now the user can give input anywhere on the monster!
Maybe you might be on an older development build, Can you please check again and let us know!

**Comment by amitsinghsutara** [comment]
As per the amount of effort required,
ben for this we won't need to put that much effort in it,
We just need to change the positions only!

**Comment by burrage** [comment]
Hey Amit I think I'm confused about what's going on... Kinda like the Arabic dropdown issue, I took the dev APK dated May 14th from the development S3 bucket and here's what I see when trying to drag to different parts on the monster:

https://curiouslearning.slack.com/files/U3MH133ME/F0737B31ADV/img_3967.mov

**Comment by amitsinghsutara** [comment]
hey @burrage ,
Unfortunately I am not able to access the  video  shared previously!

**Comment by amitsinghsutara** [comment]
Thanks @burrage  🙌
